README.md

```diff
@@ -13,25 +13,25 @@ The [creators and maintainers](/docs/GameCredits.md) are a diverse mix of softwa
 
 We encourage contributions from anybody and try to keep a warm and friendly community. 
 
-![Lumina](/docs/images/Lumina_BF_demo.png)
+![Lumina](/docs/images/Lumina_demo_2.png)
 
 ## Playing
 
-Lumina requires Unity 5.5 or higher. [download it here](https://unity3d.com/)
+We have a build that is working right now. [Download build](https://drive.google.com/drive/folders/0B0hZRv8pG5b3R21sVFI5bWF3d1E).
```

Comment:
Could you have this include the project version number? It's in the build page.
So something like "[Latest build: v0.1.0](/foo/)"

Reply:
got it